src/SML.py

```diff
@@ -1083,8 +1082,7 @@ def __init__(self, type: str, x: np.ndarray = None,
     if type == 'train':
       self.yt = DataSet(name="y", nrows=1, ncols=1)
       if yfile is not None and x is None:
-        y = np.loadtxt(yfile, skiprows=1, delimiter=",")
-        self.yt.points = y.reshape(y.points.shape[0], -1)
```

Comment:
Same issue with `self.xt.points` above. `self.y.points` is empty on initialization and so this line will return an error because it has no rows `shape[0]`

Reply:
Yes, see my reply to your previous related comment.